src/modules/ui/itemEditor/selectors.ts

```diff
@@ -0,0 +1,5 @@
+import { RootState } from 'modules/common/types'
+
+export const getState = (state: RootState) => state.ui.itemEditor
+
+export const getSelectedThirdPartyItemIds = (state: RootState) => getState(state).selectedThirdPartyItemIds
```

Comment:
I know these are simple, but a simple test for each of them would ensure that they're covered from any eventual change.

Reply:
I don't mind adding tests at all! but I'm not entirely sold on this one on particular. There's no logic, just property accessors. I'm not sure I'm convinced they'll be more useful than cumbersome.

Do you have a case in mind where they've useful? to me if we change any of these Typescript should be enough.

Reply:
We should add something in the testing standards about this.